tails_files/install.sh

```diff
@@ -1,5 +1,27 @@
 #!/bin/bash
-# SecureDrop persistent setup script for Tails 
+# SecureDrop persistent setup script for Tails
+# Usage: ./install.sh
+# parse_yaml() courtesy of a StackOverflow user:
+# http://stackoverflow.com/a/21189044
+
+set -e
+
+function parse_yaml {
+   local prefix=$2
+   local s='[[:space:]]*' w='[a-zA-Z0-9_]*' fs=$(echo @|tr @ '\034')
+   sed -ne "s|^\($s\):|\1|" \
+        -e "s|^\($s\)\($w\)$s:$s[\"']\(.*\)[\"']$s\$|\1$fs\2$fs\3|p" \
+        -e "s|^\($s\)\($w\)$s:$s\(.*\)$s\$|\1$fs\2$fs\3|p"  $1 |
+   awk -F$fs '{
+      indent = length($1)/2;
+      vname[indent] = $2;
+      for (i in vname) {if (i > indent) {delete vname[i]}}
+      if (length($3) > 0) {
+         vn=""; for (i=0; i<indent; i++) {vn=(vn)(vname[i])("_")}
+         printf("%s%s%s=\"%s\"\n", "'$prefix'",vn, $2, $3);
+      }
+   }'
+}
```

Comment:
I say it's good enough for now. Generally, I think the goal should be be to centralize the configuration management for the SecureDrop servers and the Tails workstations into one script/command/wizard.